### PR TITLE
Disable all sessions

### PR DESCRIPTION
**Pull request by YesicaNavaB**
## Summary

Disable all sessions

## Trello Card

https://trello.com/c/yeSGiWhs

### CODE REVIEW

File: app/services/users.js [hunk 1]
```diff
@@ -20,6 +20,12 @@ const upsert = userData =>
       throw error.databaseError(err.message);
     });
 
+const update = (dateToken, email) =>
+  User.update({ dateToken }, { where: { email } }).catch(err => {
+    logger.error(`Could not update user: ${err}`);
```
Comment (jersonlopez) [inline app/services/users.js]: which user? what do you think about put the email in the log?

File: app/services/users.js [hunk 2]
```diff
@@ -28,12 +34,22 @@ exports.getUser = email =>
...
+exports.validateToken = ({ email, iat }) =>
+  User.findAndCountAll({
+    where: { email },
+    attributes: ['dateToken']
+  })
     .then(result => {
       if (result.count !== 1) {
         throw error.validateTokenError('invalid Token ');
       }
+      if (result.count === 1) {
```
Comment (jersonlopez) [inline app/services/users.js]: unnecessary condition, everything you put out of the first condition will be executed if this fails

File: test/albums.test.js [hunk 3]
```diff
@@ -27,7 +27,8 @@ describe('album purchase', () => {
       lastName: 'nava',
       email: 'yesica@wolox.co',
       password: 'shdfgs345',
-      administrator: true
+      administrator: true,
+      dateToken: 1565721770
```
Comment (jersonlopez) [inline test/albums.test.js]: This field cannot be null? check it in the rest of the tests

**Review by jersonlopez — CHANGES_REQUESTED** [review]

File: app/services/users.js (hunk 1 above)
Comment (YesicaNavaB) [reply app/services/users.js]: ok!

File: app/services/users.js (hunk 2 above)
Comment (YesicaNavaB) [reply app/services/users.js]: ok!

File: test/albums.test.js (hunk 3 above)
Comment (YesicaNavaB) [reply test/albums.test.js]: ok!

**Review by jersonlopez — APPROVED** [review]

**Pull request merged**